hardware_interface/include/eps.h

```diff
@@ -58,36 +58,34 @@ typedef struct __attribute__((packed)) {
 struct __attribute__((packed)) eps_instantaneous_telemetry {
     uint8_t cmd; //value 0
     int8_t status; //0 �on success
+    double timestampInS; // not sure

```

Comment:
Are we sure now?

Reply:
Oops! Yeah, and it's precise to milliseconds. Will check the comments again. We had an interesting conversation about this in the eps PR, btw.